test/jasmine/tests/gl_plot_interact_test.js

```diff
@@ -18,6 +19,10 @@ var selectButton = require('../assets/modebar_button');
 describe('Test plot structure', function() {
     'use strict';
 
+    beforeEach(function() {
+        jasmine.addMatchers(customMatchers);
```

Comment:
The documentation on it says it should be in `beforeEach` blocks because it's torn down and rebuilt between specs, but there may be an unofficial way to have it bound globally in the tests.